hrui/hrui.php

```diff
@@ -29,6 +29,7 @@
 
 function hrui_civicrm_pageRun($page) {
   CRM_Core_Resources::singleton()->addStyleFile('org.civicrm.hrui', 'css/hrui.css');
+  CRM_Core_Resources::singleton()->addStyleFile('org.civicrm.hrui', 'css/contact.css');
```

Comment:
like this , contact.css will be included in all civicrm pages ... to specify it to contact page only you can move this line to be inside  if ($page instanceof CRM_Contact_Page_View_Summary)  statement , like this   :

```php
if ($page instanceof CRM_Contact_Page_View_Summary) {
    CRM_Core_Resources::singleton()
      ->addScriptFile('org.civicrm.hrui', 'js/hrui.js')
      ->addSetting(array('pageName' => 'viewSummary'))
      ->addStyleFile('org.civicrm.hrui', 'css/contact.css');
    //set government field value for individual page
    $contactType = CRM_Contact_BAO_Contact::getContactType(CRM_Utils_Request::retrieve('cid', 'Integer'));
    if ($contactType == 'Individual') {
      $hideGId = civicrm_api3('CustomField', 'getvalue', array('custom_group_id' => 'Identify', 'name' => 'is_government', 'return' => 'id'));
      CRM_Core_Resources::singleton()
        ->addSetting(array(
          'cid' => CRM_Utils_Request::retrieve('cid', 'Integer'),
          'hideGId' => $hideGId));
    }
  }
```